src/status_im/contexts/wallet/create_account/new_keypair/keypair_name/view.cljs

```diff
@@ -9,12 +10,20 @@
     [utils.re-frame :as rf]))
 
 (def keypair-name-max-length 15)
+(defn has-emojis? [s] (boolean (re-find utils.emojilib/emoji-regex s)))
```

Comment:
do we have a central validation for all these things in the wallet? 🤔 
cc @briansztamfater @smohamedjavid @ulisesmac  @shivekkhurana 
Seems like there is a lot of common pieces we need for account name generation, saved address etc

Reply:
AFAIK no, but yeah, we should

Reply:
We have a validators namespace. Moved them there `status-im.common.validators`